Source/Blazorise/ComponentDisposer.cs

```diff
@@ -95,6 +95,11 @@ private IList<object> LoadServiceProviderDisposableList()
 
         #region Properties
 
+        /// <summary>
+        /// Indicates if the current app is running in webassembly mode.
+        /// </summary>
+        protected bool IsWebAssembly => jsRuntime is IJSInProcessRuntime;
```

Comment:
Agreed with just reverting it back and make it so we disable this for WASM so we can stabilize the release asap.
Why this runtime based check tough? 
Microsoft provides an utility check based on a symbol : 
`OperatingSystem.IsBrowser()`
![image](https://user-images.githubusercontent.com/22283161/193127918-43265fa0-6fbd-44b8-a8b1-3605bffccaab.png)

Reply:
I didn't know about this, that's why 🤣